src/models/request-presentation.ts

```diff
@@ -38,6 +38,14 @@ export class RequestPresentation {
               proof: anchor.proofCid,
             },
           },
+          anchorCommit: {
```

Comment:
Can you add a comment on line 33, for the `anchorRecord` field: "// TODO: Remove this backwards compatibility field"

Reply:
Done, TODO comment added for anchorRecord